bin/docker-setup.sh

```diff
@@ -14,33 +14,32 @@ redirect_output() {
     fi
 }
 
-# --user xfs forces the wordpress:cli container to use a user with the same ID as the main wordpress container. See:
-# https://hub.docker.com/_/wordpress#running-as-an-arbitrary-user
 cli()
 {
 	INTERACTIVE=''
 	if [ -t 1 ] ; then
 		INTERACTIVE='-it'
 	fi
-	redirect_output docker run $INTERACTIVE --env-file default.env --rm --user xfs --volumes-from $WP_CONTAINER --network container:$WP_CONTAINER wordpress:cli "$@"
+
+	redirect_output docker exec $INTERACTIVE --env-file default.env --user www-data $WP_CONTAINER "$@"
 }
 
 set +e
 # Wait for containers to be started up before the setup.
-# The db being accessible means that the db container started and the WP has been downloaded and the plugin linked
+# The db being accessible means that the db container started and the WP has been downloaded and the plugin linked
 cli wp db check --path=/var/www/html --quiet > /dev/null
 while [[ $? -ne 0 ]]; do
 	echo "Waiting until the service is ready..."
 	sleep 5
-	cli wp db check --path=/var/www/html --quiet > /dev/null
+	cli db check --path=/var/www/html --quiet > /dev/null
```

Comment:
Looks like you missed `wp`. It should be `cli wp db check --path=/var/www/html --quiet > /dev/null`

Reply:
Fixed it [here](https://github.com/Automattic/woocommerce-payments/pull/9256/commits/e6d54ea6febccf549c0ec823b95339dbd2bdf83e).